mannequin:
    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    # Position Controllers ---------------------------------------
    head_tilt:
      type: effort_controllers/JointPositionController
      joint: neck_joint
      pid: {p: 50.0, i: 0.1, d: 3.0}
    
    left_shoulder_roll:
      type: effort_controllers/JointPositionController
      joint: left_shoulder_roll_joint
      pid: {p: 50.0, i: 0.1, d: 3.0, antiwindup: true}
      antiwindup: true 
    left_shoulder_pitch:
      type: effort_controllers/JointPositionController
      joint: left_shoulder_pitch_joint
      pid: {p: 50.0, i: 0.1, d: 3.0, antiwindup: true}
      antiwindup: true 
    left_wrist_flexion:
      type: effort_controllers/JointPositionController
      joint: left_wrist_flexion_joint
      pid: {p: 20.0, i: 0.3, d: 5.0, antiwindup: true}
      antiwindup: true 
      
    muscle:
      type: effort_controllers/JointPositionController
      joint: muscle_left_upper_arm_link_mid_joint
      pid: {p: 20.0, i: 0.3, d: 5.0, antiwindup: true}
    
      
      
    right_shoulder_roll:
      type: effort_controllers/JointPositionController
      joint: right_shoulder_roll_joint
      pid: {p: 50.0, i: 0.3, d: 3.0, antiwindup: true}
      antiwindup: true 
    right_shoulder_pitch:
      type: effort_controllers/JointPositionController
      joint: right_shoulder_pitch_joint
      pid: {p: 50.0, i: 0.3, d: 3.0, antiwindup: true}
      
    right_elbow_flexion:
      type: effort_controllers/JointPositionController
      joint: right_elbow_flexion_joint
      pid: {p: 50.0, i: 0.1, d: 3.0, antiwindup: true}
    right_wrist_flexion:
      type: effort_controllers/JointPositionController
      joint: right_wrist_flexion_joint
      pid: {p: 20.0, i: 0.3, d: 5.0, antiwindup: true}
      
      
    left_knee:
      type: effort_controllers/JointPositionController
      joint: left_knee_joint
      pid: {p: 200.0, i: 0.5, d: 1.0, antiwindup: true}
    left_ankle:
      type: effort_controllers/JointPositionController
      joint: left_ankle_joint
      pid: {p: 100.0, i: 0.5, d: 1.0, antiwindup: true}
      
    right_knee:
      type: effort_controllers/JointPositionController
      joint: right_knee_joint
      pid: {p: 200.0, i: 0.5, d: 1.0, antiwindup: true}
    right_ankle:
      type: effort_controllers/JointPositionController
      joint: right_ankle_joint
      pid: {p: 100.0, i: 0.5, d: 1.0, antiwindup: true}

